api/openapi/specs/api.yaml

```diff
@@ -172,6 +172,41 @@ paths:
           $ref: '#/components/responses/BadRequest'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /meshes/{mesh}/{serviceType}/{serviceName}/_resources/hostnames:
```

Comment:
I'm not sure about this path. What `_resources` stand for?

@lahabana any input here?

Reply:
_resources are meant to map to other resources. For example you get /_resources/dataplanes or /_resources/policies here this doesn't look right.

IMO what you want here is: `/meshes/{mesh}/{serviceType}/{serviceName}/_hostnames`